README.md

```diff
@@ -1,24 +1,144 @@
-# README
+# Movie Biskop
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The project is based on an app  to book a Movie . You can add and delet and see detail of each movie.
 
-Things you may want to cover:
+## Database Structure:
 
-* Ruby version
+![image](</img/movie-app.drawio%20(1).png>)
 
-* System dependencies
+## Project Documentation 📄
 
-* Configuration
+- Live demo [link](https://exquisite-marigold-3a82c1.netlify.app/)👈
+- API deployment [link](https://moviebiskop.herokuapp.com/api/v1/movies)👈
+- Front-end GitHub repository [link](https://github.com/bamsi/movie-booking-app-frontend)👈
 
-* Database creation
+## Kanban Board
 
-* Database initialization
+### Kanban Board [link](https://github.com/HSMathebula/movie-booking-app-back-end/projects/1)
 
-* How to run the test suite
+### Team members: 3 (Ilham Bouaik, Haji Bamsi, Happen Mathebula )
 
-* Services (job queues, cache servers, search engines, etc.)
+## Built With 🛠️
 
-* Deployment instructions
+- Programming Languages: Ruby, SQL.
+- Framework: Ruby on Rails.
+- Database: PostgreSQL.
+- Code Editor: VS Code.
+- rswag for Decomentation
 
-* ...
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and Ruby.
+
+> You can check if Git is installed by running the following command in the terminal.
+
+```
+$ git --version
+```
+
+> Likewise for Ruby installation.
+
+```
+$ ruby --version && irb
+```
+
+To install rails, in the terminal kindly run this command:
+
+```
+$ gem install rails
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/HSMathebula/movie-booking-app-back-end.git
+```
+
+### Usage
+
+Kindly modify the files as needed.
+
+### Run locally
+
+Make your way to the correct directory by running this command:
+
+```
+$ cd movie-booking-app
+```
+
+Install the required dependencies to run the project with this command:
+
+```
+$ bundle install
+```
+
+Setup the database, run:
+
+```
+$ rails db:create
+$ rails db:migrate
+$ rails db:seed
+```
+
+Then run it in your browser with this command:
+
+```
+$ rails server
+```
+
+### Run tests
+
+To run tests, please run this command:
+
+```
+$ rspec
```

Comment:
- [ ] I ran the tests according to your instructions, however, 26 of them are failing:

![image](https://user-images.githubusercontent.com/57421823/197112133-cd49be11-0112-45fa-81b7-dd508ac0a5fd.png)

Kindly check if they are passing in your env, and if they are not, please fix them to ensure your app's integrity. 💪